disnake/utils.py

```diff
@@ -50,6 +50,22 @@
 
 from .enums import Locale
 
+if sys.version_info >= (3, 11):
+    from inspect import iscoroutine as iscoroutine, iscoroutinefunction as iscoroutinefunction
```

Comment:
`iscoroutine` isn't being used anywhere, as far as I can tell, and `asyncio.iscoroutine` isn't deprecated.

Reply:
https://github.com/DisnakeDev/disnake/pull/1337/commits/3eae3ca81b89a920eb9fa2c0518da7a6ecadaca2